geoips/commandline/geoips_command.py

```diff
@@ -532,3 +532,75 @@ def add_arguments(self):
     def __call__(self, args):
         """Exectutable function for GeoipsListCommon Class. Not implemented."""
         pass
+
+
+class CommandClassFactory:
```

Comment:
I don't think we should change this now because there are dependent PRs, but I think this should just be a function rather than a class. It is used in a context where it is just used once, then thrown away, so the statefulness doesn't gain us anything and makes it a little unintuitive.

I don't think the change will be difficult. It would just be a matter of removing `self` and returning `classes` rather than accessing them via an attribute.

Reply:
See #708

Reply:
Thanks for the high praise! Thinking about this after writing it; I agree that it should just be a function rather than a class. I'll make that update at a later point. Thanks for reviewing and creating that issue. Resolving for now.